tpcx_bb/xbb_tools/utils.py

```diff
@@ -341,16 +348,13 @@ def add_empty_cli_args(args):
 
 
 def tpcxbb_argparser():
-    if os.environ.get("tpcxbb_benchmark_sweep_run") == "True":
-        import benchmark_runner
-        import importlib.resources as pkg_resources
+    import benchmark_runner
```

Comment:
Will this succeed in all directories? Is `benchmark_runner` globally visible?

Reply:
Can't seem to delete comments at the moment. But do we actually need these two imports?

Reply:
It succeeds from the tpcx-bb/tpcx_bb directory as well as from individual query directories.

Removed unused import.

Reply:
Do we need the `benchmark_runner` import? I don't see it referenced anywhere